### PR TITLE
fix links, add information to docs raised from QA

### PR DESCRIPTION
**Pull request by nvkevlu**
- Fix links after some renaming in docs restructure
- add information on new changes to the deploy_map
- add to FAQ item from QA about aborting server after corrupted snapshot
- add information about permissions for super user for Overseer commands

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -363,13 +363,10 @@ Overall training flow related questions
...
-#. Putting applications in the transfer folders without using the upload_app command or forgetting to delete the models
-   folder inside, a mysterious error may occur when running the deploy_app command because the application folder is too
-   large to be uploaded and that causes timeout.
-#. Please don't start a new training run or start a new app before the previous application is fully stopped. Users
-   can do ``abort client`` and ``abort server`` before ``start_app`` for the new run.
 #. After calling ``shutdown client`` for a client running multi GPUs, a process (sub_worker_process) may remain. The
    work around for this is to run ``abort client`` before the ``shutdown`` command.
+#. If a snapshot is in a corrupted state, the server may try to restore the job and get stuck. To resolve this, delete
+   the snapshot from the location configured in project.yml for the snapshot_persistor storage, and ``abort_job`` should
+   be able to stop the job on the server.
```
Comment (YuanTingHsieh) [inline docs/faq.rst:372]: maybe we need to provide an example to this, like `/tmp/nvflare/jobs-storage`.

Or we need to explain these in the page that contains the default project.yml
Our SAs and QAs also have questions about what these locations mean.

File: docs/user_guide/job.rst [hunk 2]
```diff
@@ -110,6 +110,11 @@ folder with meta.json.
 There is only one server, and only one app can be deployed to it for the Job, so "server" can appear only once in
 the ``deploy_map``.
 
+The ``deploy_map`` cannot be empty, and "@all" when specified as a site name carries a special meaning of all
+sites to deploy to. If "@all" is used, there should be no other apps being deployed to the sites. If an empty list of
+sites is specified for an app in the ``deploy_map``, then that app is to be deployed to no sites, and no validation is
+done other than checking that the folder exists.
```
Comment (YuanTingHsieh) [inline docs/user_guide/job.rst:116]: should we add some examples here?
Allowed:

```
deploy_map: {
    "app1": ["@ALL"], "app2": []
}
```

Not allowed:

```
deploy_map: {
    "app1": ["@ALL"], "app2": ["site-1"]
}
```
Not allowed:
```
deploy_map: {}
```

**Comment by chesterxgchen** [comment]
In general, LGTM, but I like Yuan-ting's suggestions.

**Review by YuanTingHsieh — APPROVED** [review]
Thanks, LGTM

**Pull request merged**